src/quality/Quality.tsx

```diff
@@ -17,9 +17,12 @@ export const Quality = ({
   pagination,
   heightenedCell,
   smallSize = true,
+  showReferencedColumns = true,
```

Comment:
It may be better tot add "Only" => "showReferencedColumnsOnly" - because we are always showing the reference columns but sometimes even those that are not referenced.

BTW: The set of referenced columns may contain some columns which are not in the actual row sample (and columns list). Is that ok with the code?